app/controllers/idv_controller.rb

```diff
@@ -1,7 +1,30 @@
 class IdvController < ApplicationController
-  def p1
+  def register_user(args = nil)
+    agent = Proofer::Agent.new(vendor: :mock)
+    app_vars = params.slice(:first_name, :last_name, :dob)
+    applicant = Proofer::Applicant.new(app_vars)
+    session[:resolution] = agent.start(applicant)
+    session[:question_number] = 0
+    render 'question', locals: {quest: session[:resolution].questions[session[:question_number]]}
   end
```

Comment:
I know we've been using custom controller method names, but I'd like to try sticking with just the basic RESTful CRUD actions. This would mean creating a separate controller called `Idv::UserRegistrationsController`, and putting the registration code in a `create` method since we are creating a new user registration. And instead of the `index` action,  the registration entry point would be in the `new` action in that same `UserRegistrationsController`.